coretemp.json

```diff
@@ -20,5 +16,22 @@
             "Core Temp.exe",
             "Core Temp"
         ]
-    ]
+    ],
+    "persist": [
+        "CoreTemp.ini"
```

Comment:
If `CoreTemp.ini` doesn't already exist in the .zip file, it needs to be created in a `post_install`